src/js/elements/header/current_view/current_view.js

```diff
@@ -9,6 +9,7 @@ export class CurrentView extends Component {
     }
 
     checkAndCreateDropdown(viewsArr, currentViewData) {
+        console.log({currentViewData})
```

Comment:
@emre2038 Can you remove this console.log

Reply:
done